providers/providers_manager.go

```diff
@@ -25,6 +25,11 @@ import (
 // keys stored in the data store.
 const ProvidersKeyPrefix = "/providers/"
 
+// ProviderAddrTTL is the TTL of an address we've received from a provider.
+// This is also a temporary address, but lasts longer. After this expires,
+// the records we return will require an extra lookup.
+const ProviderAddrTTL = time.Minute * 30
```

Comment:
This was a var in go-libp2p, presumably for customization? Any reason you changed to const?  (I don't really feel strongly here just want the rationale written down somewhere.)

Reply:
1. It wasn't used, as far as I can see.
2. Using package-level globals for configuration is an antipattern.

No strong feelings though, happy to change it back if desired.